### ISSUE TITLE
Would you like to add local string's key to comments?

### ISSUE THREAD

**Issue by iTofu** [mac-cain13/R.swift#663]
It's so slow to open R.generated.swift, cause there are so many lines. So it's very important that I can using 'opt + click' to preview local string's comments, to get it's value. But sometime I would also want to know it's key, than I have to open R.generated.swift, it's so slow... So could you append the key to the comments? It will look likes:
```swift
/// R.generated.swift

/// Value: Sure
static func alertConfirm(preferredLanguages: [String]? = nil) -> String {
}
// ->
/// Value: Sure, Key: alert.confirm
static func alertConfirm(preferredLanguages: [String]? = nil) -> String {
}
```

**Comment by tomlokhorst** [comment]
I think this is a good idea!

**Comment by tomlokhorst** [comment]
This is implemented in version 7.0.0
